docs/router-tutorial/react-navigation-stack/react-navigation-stack.md

```diff
@@ -7,19 +7,19 @@ has_children: false
 
 # **React Navigation**
 
-React Navigation (RN) - related libraries that help you move between screens.
+React Navigation - related libraries that help you move between screens.
 
 ## **react-navigation-stack**
 
-It's a concept that can be easily understood by anyone who's ever tried the web. Routing on the web is Stack of RN.
+It's a concept that can be easily understood by anyone who's ever tried the web. Routing on the web is Stack of React-native.
```

Comment:
`React-native` is a little awkward. Use `React Native`

Reply:
Stack is the little bit different with routing. So Routing on the web is the role of React Navigation in React Native. will be better.

Reply:
Good. i fixed it